tests/test_misc.py

```diff
@@ -846,3 +847,25 @@ def test_ipv4_and_ipv6(node_factory):
         assert bind[0]['type'] == 'ipv4'
         assert bind[0]['address'] == '0.0.0.0'
         assert int(bind[0]['port']) == port
+
+
+def test_logging(node_factory):
+    # Since we redirect, node.start() will fail: do manually.
+    l1 = node_factory.get_node(options={'log-file': 'logfile'}, may_fail=True, start=False)
+    logpath = os.path.join(l1.daemon.lightning_dir, 'logfile')
+    logpath_moved = os.path.join(l1.daemon.lightning_dir, 'logfile_moved')
+
+    # FIXME: I couldn't get super(TailableProc, l1.daemon).start() to work?
```

Comment:
I found `super` to be rather obscure. This should work however:

```
TailableProc.start(l1.daemon)
```

Since it'll take the classmethod `start` from `TailableProc` and passes in `l1.daemon` as `self`